packages/telegraf/extra/telegraf.ps1

```diff
@@ -0,0 +1,81 @@
+ param (
+    [string]$pkg_inst_dpath,
+    [string]$local_ip
+ )
+
+$conf = "$pkg_inst_dpath\etc\telegraf.conf"
+Add-Content $conf '
+[global_tags]
+[agent]
+  interval = "10s"
+  round_interval = true
+  metric_batch_size = 1000
+  metric_buffer_limit = 90000
+  collection_jitter = "0s"
+  flush_interval = "10s"
+  flush_jitter = "0s"
+  precision = ""
+  debug = false
+  quiet = false
+  logfile = ""
+  hostname = ''
```

Comment:
Will these single quotes get into the target file, given the string is quoted with single quotes?

Reply:
It looks like mistyping. Should be double quotes as per my understanding. @jongiddy  can you change this or I should do it?

Reply:
@vadymzagorodko The PR author is responsible for updating pull requests.

Reply:
fixed at 321c1271f6ac1357a94ed1107caa4f40669a7d83